scripts/Packages/Package_NoLead__DFN_QFN_LGA_SON/size_definitions/dfn.yaml

```diff
@@ -774,9 +774,15 @@ TDFN-6-1EP_2.5x2.5mm_P0.65mm_EP1.3x2mm:
 
 TDFN-14-1EP_3x3mm_P0.4mm_EP1.78x2.35mm:
   device_type: 'TDFN'
-  size_source: 'https://pdfserv.maximintegrated.com/land_patterns/90-0063.PDF (T1433-2C)'
-  body_size_x: 3 +/-0.1
-  body_size_y: 3 +/-0.1
+  size_source: 'https://pdfserv.maximintegrated.com/package_dwgs/21-0137.PDF (T1433-2C)'
+  body_size_x:
+    minimum: 2.9
+    nominal: 3.0
+    maximum: 3.1
+  body_size_y:
+    minimum: 2.9
+    nominal: 3.0
+    maximum: 3.1
```

Comment:
Its a bit strange that you mix dimension stile in the same component. Could you either use exclusively the string style or dict style?